Dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM python:2.7-slim
+
+MAINTAINER Azavea
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    g++ \
+    git \
+    gfortran \
+    libatlas-base-dev \
+    libblas-dev \
+    libgeos-dev \
+    liblapack-dev \
+    libopenmpi-dev \
+    libtool \
+    openmpi-bin \
+    python-all-dev \
+    python-pip \
+    wget
+
+# Download, build, and install GDAL
+RUN cd tmp && \
+    wget http://download.osgeo.org/gdal/gdal-1.9.0.tar.gz && \
+    tar xvfz gdal-1.9.0.tar.gz && \
+    cd gdal-1.9.0 && \
+    ./configure --with-python -with-geos=yes && \
+    make && \
+    make install
+
+# Download, build, and install PROJ
+RUN cd tmp && \
+    wget http://download.osgeo.org/proj/proj-4.8.0.tar.gz && \
+    wget http://download.osgeo.org/proj/proj-datumgrid-1.5.tar.gz && \
+    tar xzf proj-4.8.0.tar.gz && \
+    cd proj-4.8.0/nad && \
+    tar xzf ../../proj-datumgrid-1.5.tar.gz && \
+    cd .. && \
+    ./configure && \
+    make && \
+    make install
+
+# Set env vars
+ENV PATH /usr/local/bin:$PATH
+ENV CPATH /usr/local/include:/usr/include/gdal:$CPATH
+ENV CPLUS_INCLUDE_PATH /usr/include/gdal
+ENV C_INCLUDE_PATH /usr/include/gdal
+ENV LIBRARY_PATH /usr/local/lib:$LIBRARY_PATH
+ENV LD_LIBRARY_PATH /usr/local/lib:$LD_LIBRARY_PATH
+RUN git clone https://github.com/dtarb/TauDEM.git /opt/taudem
```

Comment:
Can we get away with downloading the source tarball vs. cloning the repository here?

Also, I initially missed this line because it was bundled with all of the `ENV` directives. I think it might be good to keep all of the `ENV` overrides at the top, if possible.